### PR TITLE
Feature/form layout styling

### PR DESCRIPTION
**Pull request by SeanPed**

### CODE REVIEW

File: src/app/components/AddButton/AddButton.tsx [hunk 1]
```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import classes from './AddButton.module.css';
 import { GiLightBackpack } from 'react-icons/gi';
+import { Link } from 'react-router-dom';
 
 export default function AddButton(): JSX.Element {
   return (
     <div
```
Comment (derjubu) [inline src/app/components/AddButton/AddButton.tsx:8]: The div seems unneccessary. Have you tried to use the augmentad UI directly on the link?

File: src/app/components/Form/Form.module.css [hunk 2]
```diff
@@ -3,62 +3,49 @@ textarea {
...
 }
 
 .submit_button {
-  background-color: #08d;
-  border-radius: 12px;
+  background-color: #ff003c;
   border: 0;
   box-sizing: border-box;
-  color: #eee;
+  color: snow;
```
Comment (derjubu) [inline src/app/components/Form/Form.module.css:18]: Why no hex?

File: src/app/components/Form/Form.tsx [hunk 3]
```diff
@@ -1,41 +1,40 @@
...
-    <form className={classes.input_form} action="">
+    <form
+      className={classes.input_form}
+      action=""
+      data-augmented-ui="
+      tl-clip tr-clip br-clip bl-clip border"
+    >
+      <br />
+      <FormInputField tagText="Name of item:" tagColor="Blue" textRows={1} />
+      <FormInputField tagText="Item Description:" tagColor="Red" textRows={5} />
```
Comment (derjubu) [inline src/app/components/Form/Form.tsx:15]: Where are the labels?

File: src/app/components/Form/Form.tsx [hunk 4]
```diff
@@ -1,41 +1,40 @@
...
-        defaultValue={0.2}
-        min={0.2}
-        max={25}
-        step={0.2}
-      />
-      <div className={classes.spacing_div} />
-      <div className={classes.spacing_div} />
-      <div className={classes.spacing_div} />
-      <button className={classes.submit_button}>Put it in the Backpack!</button>
+      <br />
```
Comment (derjubu) [inline src/app/components/Form/Form.tsx:30]: Try to use something else than `<br />` to get some space between the elements. You could use Flex or Grid, for exapmle.

File: src/app/components/FormInputField/FormInputField.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  textRows: number;
+};
+
+export default function FormInputField({
+  tagColor,
+  tagText,
+  textRows,
+}: inputProps): JSX.Element {
+  return (
+    <div className={classes.body_container}>
```
Comment (derjubu) [inline src/app/components/FormInputField/FormInputField.tsx:15]: Maybe use `<article>` instead?

**Review by derjubu — APPROVED** [review]
Flex your div 💪 
Looks okay to me. Just some suggestions to improve the readability a bit.

File: src/app/components/AddButton/AddButton.tsx (hunk 1 above)
Comment (SeanPed) [reply src/app/components/AddButton/AddButton.tsx:8]: yep that is unfortunately afaik not working.

File: src/app/components/Form/Form.tsx (hunk 3 above)
Comment (SeanPed) [reply src/app/components/Form/Form.tsx:15]: wdym?

**Pull request merged**